.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyc
+*.db
+*.xml
```

Comment:
I'm ignoring the data files (the xml files that contain the abstract and annotations) so it doesn't get added here. We should talk about using Git LFS as the small dataset that works with this code is a pale comparison towards the full dataset (all of pubtator) that I have parsed and ready to go.